src/scss/components/_icons.scss

```diff
@@ -0,0 +1,87 @@
+/*
+   Icons
+   ========================================================================== */
+
+
+[class^='c-icon '],
+[class^='c-icon-'],
+[class*=' c-icon '],
+[class*=' c-icon-'] {
+    speak: none;
+    background-size: contain;
+    background-position: center center;
+    background-repeat: no-repeat;
+}
+
+/**
+ * Base Icon Class
+ *
+ * Used when we have a standalone icon
+ */
+
+.c-icon {
+    display: block;
+}
+
+
+/**
+ * Modifiers for icon sizes
+ *
+ * 1. c-icon--large
+ * 2. c-icon--mid
+ * 3. c-icon--small
+ */
+
+.c-icon--large {
+    display: block;
+    width: 120px;
+    height: 120px;
+}
+
+.c-icon--mid {
+    display: block;
+    width: 60px;
+    height: 50px;
+
+    @include media('>=mid') {
+        width: 70px;
+        height: 60px;
+    }
+}
+
+.c-icon--small {
+    display: inline-block;
+    width: 25px;
+    height: 25px;
+    vertical-align: top;
+}
+
+
+.c-icon--floatLeft {
+    margin-right: spacing();
+    margin-top: 1px;
+    float: left;
+}
+
+/**
+ * Tick icons
+ */
+
+.c-icon--tick {
+    background-image: inline('#{$icon-path}/general/tick.svg');
+    .no-svg & { background-image: url('#{$icon-path}/general/tick.png'); }
+}
+
+.c-icon--tick--rounded {
+    margin-right: spacing() / 2;
+    width: 13px;
+    height: 23px;
+    background-image: inline('#{$icon-path}/general/tick-rounded.svg');
+    .no-svg & { background-image: url('#{$icon-path}/general/tick.png'); }
+}
+
+.c-icon--tick--blue,
+%c-icon--tick--blue {
+    background-image: inline('#{$icon-path}/general/tick-blue.svg');
```

Comment:
I chatted with @DamianMullins about how to use SVG spriting here yesterday. Unfortunately this is blocked until [asset sharing](https://github.com/justeat/gulp-build-fozzie/issues/61) is resolved. Shall I put this PR on hold until then or would you rather get just the buttons without icons in?